templates/index.html

```diff
@@ -1,3 +1,4 @@
+{# Templates should inherit from base rather than duplicating code#}
```

Comment:
@dhilipsiva I treated index.html as my base template that I inherited in other templates such as details.html and result.html